README.md

```diff
@@ -1,18 +1,59 @@
 # canvas-lti-redirect-tool
 This is Canvas LTI redirect tool which use the CAI developed [LTI library](https://pypi.org/project/django-lti/)
 
-## Generating Django security key
+### Prerequisites
+
+To follow the instructions below, you will at minimum need the following:
+1. **[Docker Desktop](https://www.docker.com/products/docker-desktop/)**.
+1. **[Git](https://git-scm.com/downloads)**
+### Installation and Setup
+1. You need to web Proxy like Loophole or ngrok to run the application. Loophole offers custom domain
+    ```sh
+    loophole http 6000 --hostname <your-host>
+    ```
+1. Copy the `.env.sample` file as `.env`. 
+    ```sh
+    cp .env.sample .env
+1. Examine the `.env` file. It will have the suggested default environment variable settings,
+mostly just MySQL information as well as locations of other configuration files.
+
+1. Start the Docker build process (this will take some time).
+    ```sh
+    docker compose build
+    ```
+
+1. Start up the web server and database containers.
+    ```sh
+    docker compose up
+    ```
+
+1. generate Django secret using below command
 ```sh
 python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())"
 ```
 
 ## LTI install
-1. Need to run this command once in order for LTI to work
+1. Need to run this command once docker container is up in order for LTI to work. This is important step otherwise the LTI tool launch won't happen
 ```sh
  docker exec -it clrt_web /bin/bash -c \
   "python manage.py rotate_keys" 
-```  
-2. Create superuser via using `python manage.py createsuperuser', need to run a proxy like loophole or ngrok for LTI installation and login with that user.
-https://<url>/admin/
-4. Use `LTIRegistration` to configure an LTI tool
+```
+2. use the `setup/lti-config.json` for registing the LTI tool. Replace all the `{app-hostname}` with your web proxy url.  
```

Comment:
I think we need to move this step after the current Step 4 "LTIRegistration", since the LTI registration uuid value is needed in the JSON file.

And the instruction can be more specific:

1. Go to Canvas instance, choose `Developer Keys` in admin site
2. Add LTI Key
3. Choose `Paste JSON` method
4. Copy `setup/lti-config.json` content, replacing the `{app-hostname}` with your web proxy url and `<uuid:lti-registration>` with UUID value from LTI tool registration

Reply:
Done, added all the comments as suggested.